### PR TITLE
Add theme status in `wp theme get` command

### LINKED ISSUE #172: Allow --field=status option to wp theme get command
Hello,

We are already getting the status like 'active' or 'inactive' for the plugin using the command 
"wp plugin get <plugin_name> --field=status".

But to know the status for the theme using the command "wp theme get <theme> --field=status" it outputs "Error: Invalid field: status" instead of this error it will be more useful to show the status of the theme like 'active' or 'inactive' similar to plugin status.

Thanks,
Nishit Shah

### PR DESCRIPTION
**Pull request by sagarnasit**
Add theme status filed for `wp theme get` command.

Fixes #172

### CODE REVIEW

**Comment by sagarnasit** [comment]
@thrijith added a behat test

File: features/theme.feature [hunk 1]
```diff
@@ -517,3 +517,24 @@ Feature: Manage WordPress themes
...
       """
+
+  Scenario: Get status field in theme detail
+    Given a WP install
+
+    When I run `wp theme install p2`
+    Then STDOUT should not be empty
+
+    When I run `wp theme get p2 --field=status`
+    Then STDOUT should contain:
```
Comment (thrijith) [inline features/theme.feature]:
```suggestion
    Then STDOUT should be:
```
Test could be stricter.

File: features/theme.feature [hunk 2]
```diff
@@ -517,3 +517,24 @@ Feature: Manage WordPress themes
...
+       """
+       inactive
+       """
+
+    When I run `wp theme activate p2`
+    Then STDOUT should not be empty
+
+    When I run `wp theme get p2 --field=status`
+    Then STDOUT should contain:
```
Comment (thrijith) [inline features/theme.feature]:
```suggestion
    Then STDOUT should be:
```
Same as above.

**Comment by sagarnasit** [comment]
@thrijith Added changes you asked.

File: features/theme.feature [hunk 3]
```diff
@@ -517,3 +517,30 @@ Feature: Manage WordPress themes
...
       """
+
+  Scenario: Get status field in theme detail
+    Given a WP install
+
+    When I run `wp theme install p2`
+    Then STDOUT should not be empty
+
+    When I run `wp theme get p2`
+    Then STDOUT should contain:
```
Comment (thrijith) [inline features/theme.feature]: @sagarnasit Could you update this part to be something like https://github.com/wp-cli/extension-command/blob/08984ffe611894055ebcb21b3f6bd975184fa2fe/features/theme-mod.feature#L20-L22

**Comment by sagarnasit** [comment]
Not sure why the build is failing, It seems unrelated to this PR.

**Comment by schlessera** [comment]
@sagarnasit Can you merge/rebase latest master? I think the test failure should be fixed now.

**Pull request merged**